spec/models/legal_aid_application_spec.rb

```diff
@@ -361,4 +361,86 @@
       expect(legal_aid_application.opponent_other_parties).to eq [Opponent.dummy_opponent]
     end
   end
+
+  describe 'scope_limitations' do
+    context 'adding a scope_limitation' do
+      let!(:scope_limitation_1) { create :scope_limitation, :substantive }
+      let!(:scope_limitation_2) { create :scope_limitation, :delegated_functions }
+      let!(:scope_limitation_3) { create :scope_limitation }
+      let(:application) { create :legal_aid_application }
+
+      it 'adds multiple scope limitations' do
+        application.scope_limitations << scope_limitation_1
+        application.scope_limitations << scope_limitation_2
+        application.save!
+        expect(application.scope_limitations).to match_array [scope_limitation_1, scope_limitation_2]
+      end
+    end
```

Comment:
This is just testing that ActiveRecord's `has_many` works. I don't think it's needed.